app/src/main/kotlin/com/intive/picover/common/text/PicoverOutlinedTextField.kt

```diff
@@ -1,71 +1,97 @@
 package com.intive.picover.common.text
 
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.text.KeyboardActions
 import androidx.compose.foundation.text.KeyboardOptions
 import androidx.compose.material3.OutlinedTextField
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.input.ImeAction
 import androidx.compose.ui.text.input.KeyboardType
-import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.tooling.preview.datasource.LoremIpsum
+import androidx.compose.ui.unit.dp
+import com.intive.picover.common.annotation.LightDarkPreview
+import com.intive.picover.common.validator.TextValidator
+import com.intive.picover.common.validator.ValidationStatus
+import com.intive.picover.main.theme.PicoverTheme
 
 @Composable
 fun PicoverOutlinedTextField(
 	modifier: Modifier = Modifier,
 	value: String,
-	label: String,
+	labelText: String,
 	onValueChange: (String) -> Unit,
 	maxLines: Int = 1,
 	keyboardType: KeyboardType = KeyboardType.Text,
 	imeAction: ImeAction = ImeAction.Done,
 	keyboardActions: KeyboardActions = KeyboardActions(),
-	errorText: String? = null,
 	isEnabled: Boolean = true,
+	validator: TextValidator,
+	isValid: (Boolean) -> Unit = {},
 	trailingIcon: @Composable (() -> Unit)? = null,
```

Comment:
@kmaslowiec It's not set anywhere since the creation of that component, what was the origin? Looks like copy-paste

Reply:
@MarekMacko isEnabled parameter was added just in case, same as trailingIcon. It would likely be used shortly. IMO we can keep it. If that is not your question, please specify.

Reply:
Thanks for the explanation. I would suggest to not add the code if you're not sure if it will ever be used. Keep it simple ;)